contract/src/actions.rs

```diff
@@ -334,12 +335,12 @@ impl Contract {
         let mut collateral_taken_sum = BigDecimal::zero();
 
         for asset_amount in in_assets {
-            let asset = self.internal_unwrap_asset(&asset_amount.token_id);
             liquidation_account.add_affected_farm(FarmId::Borrowed(asset_amount.token_id.clone()));
             let mut account_asset = account.internal_unwrap_asset(&asset_amount.token_id);
             let amount =
                 self.internal_repay(&mut account_asset, &mut liquidation_account, &asset_amount);
             account.internal_set_asset(&asset_amount.token_id, account_asset);
+            let asset = self.internal_unwrap_asset(&asset_amount.token_id);
```

Comment:
Moving asset here, because `internal_repay` reads and writes the asset. Even though in this place we only need `extra_decimals` from the asset config, this is not critical.